loader-load-themed-styles/README.md

```diff
@@ -24,18 +24,56 @@ var css = require("@microsoft/loader-load-themed-styles!css!./file.css");
 ### Example config
 
 ``` javascript
-module.exports = {
-  module: {
-    loaders: [
-      { test: /\.css$/, loader: "@microsoft/loader-load-themed-styles!css-loader" }
-    ]
-  }
-};
+        use: [
+          {
+            loader: "@microsoft/loader-load-themed-styles",  // creates style nodes from JS strings
+            options: {
+              namedExport: 'default'
+            }
+          },
+          {
+            loader: "css-loader", // translates CSS into CommonJS
+            options: {
+              modules: true,
+              importLoaders: 2,
+              localIdentName: '[name]_[local]_[hash:base64:5]',
+              minimize: false
+            }
+          },
+          {
+            loader: 'postcss-loader',
+
+            options: {
+              plugins: function () {
+                return [
+                  require('autoprefixer')
```

Comment:
Nope - doesn't look like it.

---
In reply to: [124628930](https://github.com/Microsoft/webpack-loaders/pull/12#discussion_r124628930) [](ancestors = 124628930)